spec/models/participant_profile_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ParticipantProfile, type: :model do
+  before :each do
+    @object = create(:participant_profile)
+  end
+  let(:ect_profile) { create(:early_career_teacher_profile) }
+
+  describe "state transitions" do
+    it "should have an initial state of assigned" do
+      expect(@object).to have_state(:assigned)
+    end
+
+    it "should only have expected transitions from the initial state" do
```

Comment:
How do state changes relate to the start event?

Reply:
These are atomic participant states. For the "start" event, the expectation for the participant is to set them up initially and to make them "active". I can run you through the logic tomorrow if you want, but it's probably a bit deeper than PR comments can convey correctly.